ci/cloudbuild/builds/publish-docs.sh

```diff
@@ -21,8 +21,17 @@ source module ci/cloudbuild/builds/lib/cmake.sh
 source module ci/cloudbuild/builds/lib/features.sh
 source module ci/lib/io.sh
 
-mapfile -t FEATURE_LIST < <(features::list_full)
-read -r ENABLED_FEATURES < <(features::list_full_cmake)
+if [[ "${TRIGGER_TYPE}" == "ci" ]]; then
```

Comment:
Consider changing this to be like:
```
if [[ "${LIBRARIES}" == "all" ]]; then
```

... and setting `_LIBRARIES: all` in `publish-docs-ci.yaml`